### ISSUE TITLE
Twine warning: `long_description_content_type` missing

### ISSUE THREAD

**Issue by rrjbca** [skypyproject/skypy#103]
**Describe the bug**
When packaging for release, twine warns that `long_description_content_type` is missing.

**To Reproduce**
1. `python setup.py build sdist`
2. `twine check dist/*`
```
Checking dist/skypy-0.1rc1.tar.gz: PASSED, with warnings
  warning: `long_description_content_type` missing.  defaulting to `text/x-rst`.
```

**Fix**
Add `long_description_content_type = text/x-rst` to `setup.cfg`

**Comment by Lucia-Fonseca** [comment]
I have this issue:
`fatal: 'skypyproject' does not appear to be a git repository
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.`

**Comment by rrjbca** [comment]
Fixed by #111, closing
